src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkCustomCommandLine.java

```diff
@@ -655,12 +692,13 @@ public boolean equals(Object o) {
         return false;
       }
       VectorArg vectorArg = (VectorArg) o;
-      return features == vectorArg.features;
+      return features == vectorArg.features
+          && Objects.equal(stringificationType, vectorArg.stringificationType);
     }
 
     @Override
     public int hashCode() {
-      return Integer.hashCode(features);
+      return 31 * Integer.hashCode(features) + HashCodes.hashObject(stringificationType);
```

Comment:
Now that `stringificationType` is non-null, just call `hashCode()` directly on it instead of using `HashCodes`.